### PR TITLE
Config bullet

### PR DESCRIPTION
**Pull request by Chrede88**
TOML config now works, apart form a small problem with to many quotes in text waves.

### CODE REVIEW

File: ScanController.ipf [hunk 1]
```diff
@@ -374,105 +374,255 @@ end
...
+			str = stringfromlist(i,key,":")
+		endif
+		index = strsearch(TOMLstr,str,old_index)
+		if(index==-1 || index<old_index)
+			return ""
+		endif
+		old_index=index+strlen(str)
+	endfor
+	val_start = index+strlen(str)+1
+	val_end = strsearch(TOMLstr,num2char(13),val_start) // look for \r
```
Comment (nikhartman) [inline ScanController.ipf]: I think this is going to break if you happen to be using Windows. The line endings are OS specific. Look for "\r" or "\n" might be right?

Comment (Chrede88) [reply ScanController.ipf]: I checked! if you don't specific /T=<term> in freadline igor adds \r by default. So it should be consistent across platforms.

Comment (nikhartman) [reply ScanController.ipf]: Good work, Igor.

**Comment by Chrede88** [comment]
Config files looks like this now:
```# sc1544430811.toml

[waves]
raw=["g1x","g1y"]
calc=["",""]

[checkboxes]

	[checkboxes.record]
	raw=[false,false]
	calc=[false,false]

	[checkboxes.plot]
	raw=[false,false]
	calc=[false,false]

	[checkboxes.async]
	async=]

	[checkboxes.history]
	raw=true
	calc=true

[scripts]
raw=["readSRSx(srs1)","readSRSy(srs1)"]
calc=["",""]

logstring="GetSRSStatus(srs1);"

filenum=0```

**Comment by Chrede88** [comment]
There might still be an issue with the logstring entry! I'll take a look at that tommorow. I'm pretty sure that the notifySlack thing will break it.

**Comment by Chrede88** [comment]
But let's just merge for now.

**Comment by nikhartman** [comment]
I dumbed down the slack thing a lot just now. It might be ok. 

I think there are a few ways to make this a little more robust. Let's keep working here for now.

**Pull request merged**